resources.go

```diff
@@ -36,6 +36,14 @@ const (
 	samlMod   = "Saml"   // the Saml module
```

Comment:
Not part of this PR - but aren't these module names supposed to be lowercase? Now, we get Node.js modules called "Saml" etc., which doesn't sound right to me.

Reply:
Ok, let's you and I get together later this week and fix these up and make a new release